test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresCapability.attributes.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
The XML files should also be renamed I think - so `RequiresViaXml.attributes.xml`... and similarly below.